openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/AbstractStream.java

```diff
@@ -48,8 +49,15 @@ public void release() {
 
 	public abstract void release(boolean remove);
 
-	public static WebRtcEndpoint createWebRtcEndpoint(MediaPipeline pipeline, Boolean recv) {
+	public static WebRtcEndpoint createWebRtcEndpoint(MediaPipeline pipeline, Boolean recv,
```

Comment:
the code will be cleaner if this `Boolean` will be `boolean` :)

Reply:
I can change it. Wasn't my change to introduce this parameter.

Reply:
That is actually not that easy.
The null value is used as a 3rd option to pass in.
In KTestStream::startTestRecording it passes _explicitly_ null into the method so that the stream is not marked as **neither** recv or send only.